workshops/views.py

```diff
@@ -1112,6 +1112,9 @@ class EventCreate(LoginRequiredMixin, PermissionRequiredMixin,
     form_class = EventForm
     template_name = 'workshops/event_create_form.html'
 
+    def get_initial(self):
+        return {'invoice_status': 'not-invoiced'}
+
```

Comment:
I tried that before, it ends up breaking a test. 
`tesr/test_event.py L:50`
```
    def test_get_uninvoiced_events(self):
        """Test that the events manager can find events that owe money"""

        uninvoiced_events = Event.objects.uninvoiced_events()

        # There should be as many as there are strictly future events.
        assert len(uninvoiced_events) == self.num_uninvoiced_events

        # Check that events with a fee of zero or None are still on this list
        assert any([x for x in uninvoiced_events if not x.admin_fee])
```
It asserts that the number of uninvoiced events are the same as the number of future event, which breaks after changing the default.
Moreover, `unknown` seems to be the right default for the `invoice_status` rather than `not-invoiced` when the field `invoice_status` is not specified.